shared/gh/js/controllers/gh.api.util.js

```diff
@@ -215,25 +215,30 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
         // Loop over the terms
         _.each(terms, function(term) {
             if (term.name === termName) {
-                // The number of milliseconds in one week
-                var ONE_WEEK = 1000 * 60 * 60 * 24 * 7;
-                // The number of milliseconds in one day
-                var ONE_DAY = 1000 * 60 * 60 * 24;
                 // Get the date on which the term starts
                 var termStartDate = new Date(term.start).getTime();
 
                 // Calculate the week offset in milliseconds
-                var weekOffset = weekNumber * ONE_WEEK;
+                var weekOffset = (weekNumber - 1) * PERIODS['week'];
```

Comment:
It would give back a date, but outside of term. e.g.

```javascript
> require('gh.core').api.utilAPI.getDateByWeekAndDay('michaelmas', 1, 2)
- Tue Sep 30 2014 01:00:00 GMT+0100 (BST)
> require('gh.core').api.utilAPI.getDateByWeekAndDay('michaelmas', 0, 2)
- Tue Oct 07 2014 01:00:00 GMT+0100 (BST)
```